src/diffusers/loaders.py

```diff
@@ -322,9 +321,16 @@ def load_attn_procs(self, pretrained_model_name_or_path_or_dict: Union[str, Dict
         attn_processors = {}
         non_attn_lora_layers = []
 
-        is_lora = all("lora" in k for k in state_dict.keys())
+        is_lora = all(("lora" in k or k.endswith(".alpha")) for k in state_dict.keys())
         is_custom_diffusion = any("custom_diffusion" in k for k in state_dict.keys())
 
+        for k in state_dict:
+            new_k = k.replace("lora_unet_", "")
+            new_k = new_k.replace("lora_te_", "")
+            if "lora" not in new_k:
+                pass
```

Comment:
@patrickvonplaten at this point in the code, we actually don't expect any key in the `state_dict` to have `lora_unet_` or `lora_te_` since we `diffusers` format doesn't have it. In the Kohya conversion utility, we make sure these identifiers are replaced w.r.t the diffusers format. 

Could I get an explanation on why this block is here?

Reply:
Ah I think we can remove this - think it was either already there or I just added for testing